malevo/swagger.yml

```diff
@@ -0,0 +1,174 @@
+swagger: '2.0'
+info:
+  description: Malevo API
+  version: 0.1.0
+  title: Malevo API
+#host: localhost
+basePath: /api/malevo
+tags:
+  - name: class
+    description: Everything about predicted and ground truth classes
+  - name: images
+    description: Everything about images
+schemes:
+  - http
+paths:
+  /class/sampleImages:
+    get:
+      tags:
+        - class
+      summary: Representative sample of images
+      description: >-
+        Returns a list of representative images for that matches all given class
+        ids
+      produces:
+        - application/json
+      parameters:
+        - name: dataset
+          in: query
+          description: Phovea dataset id
+          required: true
+          type: string
+        - name: epoch
+          in: query
+          description: Selected epoch id
+          required: true
+          type: integer
+        - name: groundTruthClassId
+          in: query
+          description: Ground truth class id
+          required: true
+          type: integer
+        - name: predictedClassId
+          in: query
+          description: Predicted class id
+          required: true
+          type: integer
+        - name: numCount
+          in: query
+          description: Number of images
+          required: false
+          default: 100
+          minimum: 1
+          type: integer
+      responses:
+        '200':
+          description: >-
+            Successful operation returns a list of image ids
+          schema:
+            type: array
+            items:
+              type: integer
+        '400':
+          description: Invalid input value
+
+  /class/crossEntropy:
```

Comment:
I would group crossEntropy under a separate tag `measures` as it's not really related to one class but specific for an img id and a specific epoch